test/remote/device_tests_refresh_tokens.js

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Is this largely a copy/paste of the existing `device_tests.js` file?  I wonder if we can refactor it to avoid duplication and share the core test logic between `sessionToken` and `refreshToken` cases, to avoid divergence over time.

Reply:
maybe later....